configuration/cdap-site.xml

```diff
@@ -138,283 +169,257 @@
+    </value-attributes>
   </property>
 
-  <!-- Stream handles -->
-
   <property>
-    <name>stream.flume.port</name>
-    <value>10004</value>
+    <name>app.temp.dir</name>
+    <value>/tmp</value>
   </property>
 
   <property>
-    <name>stream.flume.threads</name>
+    <name>scheduler.max.thread.pool.size</name>
     <value>100</value>
+    <description>Size of the scheduler thread pool</description>
+    <value-attributes>
+      <type>int</type>
+      <overridable>false</overridable>
+    </value-attributes>
   </property>
 
   <property>
-    <name>stream.base.dir</name>
-    <value>/streams</value>
-    <description>The directory root for all stream files, relative to the HDFS namespace</description>
-  </property>
-
-  <property>
-    <name>stream.partition.duration</name>
-    <value>3600000</value>
+    <name>workflow.token.max.size.mb</name>
+    <value>30</value>
     <description>
-      The default duration of a stream partition in milliseconds
+      Maximum allowed size of a Workflow Token, in megabytes; if the
+      workflow token exceeds this size, no further updates are allowed
     </description>
+    <value-attributes>
+      <type>int</type>
+      <overridable>false</overridable>
+    </value-attributes>
   </property>
 
+  <!-- Audit Configuration -->
+
   <property>
-    <name>stream.index.interval</name>
-    <value>10000</value>
+    <name>audit.enabled</name>
+    <value>true</value>
     <description>
-      Default time interval in milliseconds for emitting new index entry in
-      stream file
+      Determines whether to publish audit messages to Apache Kafka
     </description>
+    <display-name>Publish Audit Logs</display-name>
+    <value-attributes>
+      <type>boolean</type>
+      <overridable>false</overridable>
+    </value-attributes>
   </property>
 
   <property>
-    <name>stream.event.ttl</name>
-    <value>9223372036854775807</value>
-    <description>
-      Default time to live in milliseconds (Long.MAX_VALUE) for stream
-      events
-    </description>
-  </property>
-
-  <property>
-    <name>stream.container.instance.id</name>
-    <value>0</value>
-    <description>
-      Instance ID for the stream service container; the actual value will be
-      set at runtime by the system automatically
-    </description>
-  </property>
-
-  <property>
-    <name>stream.notification.threshold</name>
-    <value>1024</value>
+    <name>audit.kafka.topic</name>
+    <value>audit</value>
     <description>
-      Size of data, in megabytes, to be ingested by a stream before a
-      notification is published
+      Apache Kafka topic name to which audit messages are published
     </description>
   </property>
 
-  <property>
-    <name>stream.size.schedule.polling.delay</name>
-    <value>600</value>
-    <description>Delay, in seconds, to poll a stream in a StreamSizeSchedule if no notification is received</description>
-  </property>
+  <!-- Datasets Configuration -->
 
```

Comment:
seems to be missing leveldb configuration. on purpose? since its not needed in distributed mode? (data.local.storage*)